src/sonic-yang-mgmt/sonic_yang.py

```diff
@@ -540,3 +540,85 @@ def find_data_dependencies (self, data_xpath):
             self.fail(e)
 
         return ref_list
+
+    """
+    get_module_prefix:   get the prefix of a Yang module
+    input:    name of the Yang module
+    output:   prefix of the Yang module
+    """
+    def get_module_prefix(self, module_name):
+        prefix = ""
+        try:
+            module = self.get_module(module_name)
+        except Exception as e:
+            self.fail(e)
+            return prefix
+        else:
+            return module.prefix()
```

Comment:
Hi Ping 

Actually, This prefix is not the one which is used in leafref. This is the prefix defined for each module, where as we need the prefix which is used in import statement. Import statement can use a name which is different then module prefix. 
For example in sonic-acl, we can have port leafref.
```
 path "/port:sonic-port/port:PORT/port:PORT_LIST/port:port_name;"
```
Here port prefix comes from import statement: 
```
	import sonic-port {
		prefix port;<<<<<<<
		revision-date 2019-07-01;
	}
```
Whereas, this API will return below prefix:
```
module sonic-port{
	namespace "http://github.com/Azure/sonic-port";
	prefix sonic-port;<<<<<<<<< [We can write any name here]

```